.prettierignore

```diff
@@ -7,4 +7,5 @@ src/components.d.ts
 src/components/*/readme.md
 src/global/core/components/*/readme.md
 convenience/generate-component/boilerplate/readme.md
+convenience/generate-component/boilerplate/component.tsx
```

Comment:
This line is required because prettier strips the apexes before the component name at line 57